Realm/RLMAccessor.mm

```diff
@@ -549,40 +549,40 @@ static void RLMSuperSet(RLMObjectBase *obj, NSString *propName, id val) {
     superSetter(obj, prop.setterSel, val);
 }
 
-// getter/setter for unmanaged
-static IMP RLMAccessorStandaloneGetter(RLMProperty *prop, RLMAccessorCode accessorCode) {
+// getter/setter for standalone
```

Comment:
This file still uses Standalone everywhere except this one comment. The switch in terminology should be done across the entire implementation at once or it just makes things more confusing.